fcc-test-suite-template-master/index.html

```diff
@@ -7,6 +7,7 @@
 	<title>Tribute to Steve Jobs</title>
```

Comment:
- [ ] You don't always include an explanation to why you request the changes, kindly make sure to explain why you requested changes so your comment makes more sense